src/CMakeLists.txt

```diff
@@ -26,6 +26,19 @@ configure_package_config_file(
     "${PROJECT_BINARY_DIR}/${CMAKE_FILES_DIRECTORY}/libappimageConfig.cmake"
     INSTALL_DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/libappimage
 )
+
+configure_file(
```

Comment:
Why not use `install(...)`? Do you really need the files at config time?

Reply:
This file is also required at config time, it's function is to allow using the produced binaries from other projects without the need to install it.